full_snapshot.proto

```diff
@@ -291,6 +291,8 @@ message CollectorConfig {
 
   string crunchy_bridge_cluster_id = 75;
 
+  string aiven_service_id = 79;
```

Comment:
Per my comment on the collector PR, we may want to track the project ID separately here, since the Aiven hostname consists of both the service ID and the project ID (combined with a dash).